indexer/ai/src/main/java/com/google/cloudsearch/ai/StandardSkillEntityExtraction.java

```diff
@@ -0,0 +1,200 @@
+package com.google.cloudsearch.ai;
+
+import com.google.cloud.language.v1.*;
+import com.google.common.collect.Multimap;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class StandardSkillEntityExtraction implements AISkill {
+
+    private String aiSkillName = "";
+    private List<OutputMapping> outputMappings = new ArrayList<OutputMapping>();
```

Comment:
Yes, even that can be done, just kept it because output mapping is one of the main features of the skill config, thought it would be more easy to read if we have getMapping and set Mapping etc.